app/Libraries/TeamSpeak.php

```diff
@@ -376,7 +376,7 @@ public static function checkClientIdleTime(TeamSpeak3_Node_Client $client, Accou
     {
         $idleTime = floor($client['client_idle_time'] / 1000 / 60); // minutes
 
-        if ($member->hasPermissionTo('teamspeak/idle/permanent')) {
```

Comment:
Can we go further with this check? At present, they could not be on a UK position.

Reply:
I can't find anything to suggest this? It should just check if they have an active ATC session regardless of where it is?

Reply:
I was just taking that from the JIRA story

Reply:
Ah wording subtleties... I will amend to user ;)